packages/notebook-extension/schema/tracker.json

```diff
@@ -47,6 +47,21 @@
       "keys": ["R"],
       "selector": ".jp-Notebook:focus"
     },
+    {
+      "command": "notebook:move-cell-down",
+      "keys": ["Ctrl J"],
+      "selector": ".jp-Notebook:focus"
+    },
+    {
+      "command": "notebook:move-cell-up",
+      "keys": ["Ctrl K"],
+      "selector": ".jp-Notebook:focus"
+    },
```

Comment:
The suggested change below would bring these commands in line with what the extensions currently do. Although having a command to move cells up and down in jupyter command mode would also be awesome. Maybe we could add both?

```suggestion
    {
      "command": "notebook:move-cursor-down",
      "keys": ["Ctrl J"],
      "selector": ".jp-Notebook.jp-mod-editMode"
    },
    {
      "command": "notebook:move-cursor-up",
      "keys": ["Ctrl K"],
      "selector": ".jp-Notebook.jp-mod-editMode"
    },
```

Reply:
> having a command to move cells up and down in jupyter command mode would also be awesome.

this applies in `jupyter command mode`. Maybe you mean in `VIM command mode`?

Reply:
Oh yeah, my bad for not being more precise with my terminology. Currently the extensions enable you to shift the cursor up or down to adjacent cells using `ctrl-j/k` in either `vim-insert` or `vim-command` modes. Though I think having these keybinds do something `jupyter-command` mode is also a valuable addition. Like so:

![shifting-cursor](https://user-images.githubusercontent.com/10111092/88583006-9fb09400-d01d-11ea-93e5-e780d9a20af8.gif)